overlord/snapstate/snapmgr.go

```diff
@@ -420,6 +435,9 @@ func (m *SnapManager) doDiscardSnap(t *state.Task, _ *tomb.Tomb) error {
 			newSeq = append(newSeq, si)
 		}
 		snapst.Sequence = newSeq
+		if snapst.Current == ss.Revision {
```

Comment:
We can easily bring it back if we need to. I'd prefer that over keeping state we have no use for because some day we might need it.